modules/lakeformation/main.tf

```diff
@@ -0,0 +1,45 @@
+resource "aws_lakeformation_permissions" "segment_data_lake_formation_table_emr" {
+  principal = var.emr_role
+  permissions = ["ALL"]
+
+  table {
+    database_name = var.name
+    wildcard = true
+  }
+}
+
+resource "aws_lakeformation_permissions" "segment_data_lake_formation_database_emr" {
+  principal   = var.emr_role
+  permissions = ["ALL"]
+
+  database {
+    name       = var.name
+  }
+}
+
+resource "aws_lakeformation_permissions" "segment_data_lake_formation_database_creator" {
```

Comment:
Would be good to call out this block is equivalent to making the role a Database creator in Lake Formation.